pkg/config/config.go

```diff
@@ -102,6 +123,7 @@ type proxyLocalConfig struct {
 // serialize it.
 type LocalConfigInfo struct {
 	Filename         string `yaml:"FileName,omitempty"`
+	fs               fs
```

Comment:
I am usually against having attributes which allow us to test being present in the main code. But the alternative is to have a global `fs` variable which I feel is even worse.

Reply:
> I am usually against having attributes which allow us to test being present in the main code.

Agreed. This is not a good approach :-(
There has to be a better way to do this.
/hold

Reply:
I can keep it global

Reply:
One good thing about this approach is that default behavior is preserved

Reply:
> I can keep it global

I'm afraid that that would make it even worse :-(

Reply:
/hold cancel
Now we are using afero with the best approach.